app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,298 @@
+            android:drawableStart="@drawable/ic_path"
+            android:drawablePadding="8dp"
+            android:fontFamily="@font/roboto_light"
+            android:text="@string/back_text_view"
+            android:textColor="@color/for_back"
+            android:textSize="12dp"
+            app:layout_constraintStart_toStartOf="@id/guideline_start"
+            app:layout_constraintTop_toTopOf="@id/image_view_background" />
+
+        <TextView
+            android:id="@+id/age_limit_text_view"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginBottom="56dp"
+            android:background="@drawable/age_limit_rectangle"
+            android:fontFamily="@font/roboto_black"
+            android:gravity="center"
+            android:text="@string/age_limit"
+            android:textColor="@color/white"
+            android:textSize="12dp"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toTopOf="@id/name"
+            app:layout_constraintStart_toStartOf="@id/guideline_start" />
+
+        <TextView
+            android:id="@+id/name"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="254dp"
+            android:fontFamily="@font/roboto_black"
+            android:text="@string/name_of_film"
+            android:textColor="@color/text_color"
+            android:textSize="40dp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toStartOf="@id/guideline_end"
+            app:layout_constraintStart_toStartOf="@id/guideline_start"
+            app:layout_constraintTop_toTopOf="@id/image_view_background" />
+
+        <TextView
+            android:id="@+id/film_genre"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:fontFamily="@font/roboto_light"
+            android:text="@string/film_genre"
+            android:textColor="@color/film_genre_color"
+            android:textSize="14dp"
+            app:layout_constraintEnd_toStartOf="@id/guideline_end"
+            app:layout_constraintStart_toStartOf="@id/guideline_start"
+            app:layout_constraintTop_toBottomOf="@id/name" />
+
+        <ImageView
+            android:id="@+id/first_star"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@drawable/ic_star_icon"
+            app:layout_constraintBottom_toBottomOf="@id/reviews"
+            app:layout_constraintStart_toEndOf="@id/guideline_start"
+            app:layout_constraintTop_toTopOf="@id/reviews" />
+
+        <ImageView
+            android:id="@+id/second_star"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:layout_marginEnd="3dp"
```

Comment:
android:layout_marginEnd="3dp" has no sense
as third_star depends on second_star
not vice versa

and recheck all stars